src/server/migrations/20190905195534-create-roles-table.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+exports.up = function(db) {
+    return db.runSql(
+        `CREATE TABLE "roles" (
+        id SERIAL PRIMARY KEY,
+        role varchar NOT NULL CHECK (role IN ('super_admin', 'admin', 'researcher', 'facilitator', 'participant')),
+        userid int NOT NULL,
+        FOREIGN KEY(userId) REFERENCES users(id)
```

Comment:
`userid` vs `userId` vs `user_id` <-- this last one is the "sql preference" for column names

Reply:
Good to know, updated!